data/json/mapgen_palettes/wasp_palette.json

```diff
@@ -0,0 +1,144 @@
+[
+  {
+    "type": "palette",
+    "id": "wasp_palette",
+    "terrain": {
+      "x": "t_paper_brood",
+      "X": "t_paper_hard",
+      "_": "t_floor_paper_hard",
+      "*": "t_roof_paper_hard",
+      ".": "t_floor_paper_noroof",
+      "8": "t_floor_paper_noroof",
+      "R": "t_radio_tower",
+      "a": "t_railing",
+      ",": "t_metal_floor_no_roof",
+      "<": "t_stairs_up",
+      ">": "t_stairs_down",
+      ":": "t_null",
+      "#": "t_metal_floor_no_roof"
+    },
+    "monster": { "8": { "group": "GROUP_WASP_QUEEN" } },
+    "furniture": { ":": "f_cellphone_booster", "#": "f_small_satelitte_dish" },
+    "nested": { "O": { "chunks": [ "wasp_brood" ] } }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "nested_mapgen_id": "wasp_brood",
+    "//": "Empty cell",
+    "weight": 100,
+    "object": { "mapgensize": [ 1, 1 ], "rows": [ "0" ], "terrain": { "0": "t_open_air" } }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "nested_mapgen_id": "wasp_brood",
+    "//": "Egg cell.",
+    "weight": 200,
+    "object": {
+      "mapgensize": [ 1, 1 ],
+      "rows": [ "0" ],
+      "terrain": { "0": "t_floor_brood_fake" },
+      "item": { "0": { "item": "egg_wasp" } }
```

Comment:
It seems like `egg_wasp` is not defined in this this pr or defined in the `data` folder, weirdly it is mentioned here <https://github.com/CleverRaven/Cataclysm-DDA/blob/58d6d54fe588daa974a57e922f7f7810f794a831/gfx/RetroDaysTileset/tile_config.json#L2553>

Reply:
Odd re that tile_config; `egg_wasp` might be defined in a "prerequisite" PR, #47960.

Reply:
Sorry, didn't read all the important parts >.<

Reply:
In my experience, most people frequently miss those...